active-rfcs/0001-contribution.md

```diff
@@ -0,0 +1,60 @@
+- Feature Name: Contribution
+- Start Date: 2021-03-01
+- Reference Issues:
+- Implementation PR:
+
+# Summary
+
+A contribution plugin will be developed for tichi, which is mainly responsible for labeling external members' PRs with `contribution` or `first-time-contributor` labels.
+
+# Motivation
+
+We need to distinguish between internal and external contributor PRs, and prioritize help with external contributor PRs, especially first-time contributor PRs, which will allow external contributors to have a good contribution experience.
+
+# Detailed design
+
+## contribution
+
+For external contributors, we will label their PR with `contribution`.
+
+### Definition:
+
+- Label Name: `contribution`
+- Usage: External Contributors
+  - We will determine if it is an external contributor by whether it is an org member or not.
```

Comment:
Is `org member` equivalent to committer?

Reply:
Yes.

Reply:
Ok

Reply:
Sorry, it's reviewers.

Reply:
why not to use the sig json?

Reply:
Because I think even active-contributors need to get some attention.

Reply:
I remember you used to think so too. 🤣

Reply:
I mean, use the committer in the sig json as the filter condition.

Reply:
I don't think the reviewer's pr should be labeled, because reviewers are generally familiar with the collaborative process and can find others to help with reviews as quickly as possible.

Also by definition, it is more obvious and better to understand by whether it is an org member to classify whether it is a contribution or not.